sycl/plugins/unified_runtime/CMakeLists.txt

```diff
@@ -93,18 +104,20 @@ add_sycl_plugin(unified_runtime
     "ur/adapters/level_zero/ur_level_zero_program.cpp"
     "ur/adapters/level_zero/ur_level_zero_queue.cpp"
     "ur/adapters/level_zero/ur_level_zero_sampler.cpp"
-    "ur/usm_allocator.hpp"
-    "ur/usm_allocator.cpp"
-    # These below belong to Unified Runtime PI Plugin only
-    "pi_unified_runtime.hpp"
-    "pi_unified_runtime.cpp"
-    LIBRARIES
-    Threads::Threads
+  INCLUDE_DIRS
+  LIBRARIES
     UnifiedRuntime-Headers
+    #UnifiedRuntimeLoader
```

Comment:
Why UnifiedRuntimeLoader is commented out?
Usually adding commented out code is considered to be a bad practice. Can we drop this line?